server/public/model/utils.go

```diff
@@ -844,3 +844,16 @@ func filterBlocklist(r rune) rune {
 func IsCloud() bool {
 	return os.Getenv("MM_CLOUD_INSTALLATION_ID") != ""
 }
+
+func sliceToMapKey(s ...string) map[string]any {
```

Comment:
Not sure I understand the need for this other than to make the initialization side look a bit prettier :p

Reply:
Exactly that :)

Reply:
You are proper aesthete :)